### DISCUSSION TITLE
Debugging a Zeit/now.sh deployment via static export: routes-manifest.json could not be found

### DISCUSSION THREAD

**Discussion by erikdstock** [vercel/next.js#11972] [Help] [answered]
Hi folks - I'm not sure if this is a zeit configuration issue or a next export issue, so please excuse me if I'm posting in the wrong place.

1. I have a [PR branch](https://github.com/uglycaucus/uglycauc-us/pull/12) that is working fine with both `next dev` and `next build`/`export`/`serve`. However it is not working on zeit. My configuration is pretty straightforward:
![image](https://user-images.githubusercontent.com/9088720/79570978-606ec080-8088-11ea-91bc-9fdccc5350d2.png)

2. I have some new static build config in one page here:
https://github.com/uglycaucus/uglycauc-us/pull/12/files#diff-eea85ff38f6fd6ea96f3935a84157df1R30-R42

3. I have tried this with both next `9.3.0`, `9.3.4` and `9.3.5`. All of them fail to deploy on zeit with `Error: A routes-manifest.json couldn't be found. This could be due to a failure during the build.`. 

4. Some variance that might be a red herring: However, with the later patch versions the build takes a very long time - going from < 2 minutes to over 20. These now.sh log lines are separated by 3 seconds using 9.3.0 but over 20 minutes with .4/.5. I get similar results locally:
```
07:42:41.127  Automatically optimizing pages...
08:03:05.985  Page  (...the exported page map)
```

Here is the end of the failing preview deployment logs:
```
08:03:07.859  
> using build directory: /zeit/2def63aa/.next
08:03:07.862  
  copying "static build" directory
08:03:07.865  
> No "exportPathMap" found in "next.config.js". Generating map from "./pages"
08:03:07.866  
  launching 3 workers
08:03:07.866  
Exporting (0/4)
08:03:07.866  
  copying "public" directory
08:03:07.975  
...
Export successful
08:03:08.103  
Error: A routes-manifest.json couldn't be found. This could be due to a failure during the build
08:03:08.103  
    at Object.getRoutesManifest (/zeit/1bc6bc026129a8fb/.build-utils/.builder/node_modules/@now/next/dist/index.js:6113:15)
08:03:08.103  
    at async Object.module.exports.178.exports.build (/zeit/1bc6bc026129a8fb/.build-utils/.builder/node_modules/@now/next/dist/index.js:3764:28)
```

**Comment by Timer** [comment, marked as answer]
This is a misconfiguration! Please disable overriding the output directory: the output directory is still `.next` (dist dir).

Next.js automatically knows that you ran an export.
